api/utils/utils.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Nit: I think something like `bool.go` would be more appropriate name for this file. Unless you intend to turn it into a collection of random utility methods later.

Reply:
That was my original intention, but I don't think that'll happen after all.